src/components/popover/bl-popover.ts

```diff
@@ -219,7 +219,12 @@ export default class BlPopover extends LitElement {
 
   private _handlePopoverShowEvent(event: Event) {
     if (event.target !== this) {
-      this.hide();
+      const { parentElement } = event.target as HTMLElement;
+      const hasPopoverParent = parentElement?.closest("bl-popover") === this;
```

Comment:
Does this work with a 3rd level nested popover? Maybe we could consider something like:

```ts
const isNestedPopover = this.contains(parentElement);
```

Reply:
Totally missed that. That was a null check before. Thanks.